apps/uvc-app/tof-sdk-interface.cpp

```diff
@@ -137,12 +137,18 @@ void handleClientRequest(const char *in_buf, const size_t in_len,
     }
 
     case uvc_payload::FunctionName::GET_AVAILABLE_FRAME_TYPES: {
-        std::vector<aditof::DepthSensorFrameType> frameTypes;
+        std::vector<std::string> frameTypes;
+        std::vector<aditof::DepthSensorFrameType> availableFrameTypes;
         auto depthSensorFrameTypesMsg =
             response.mutable_available_frame_types();
 
         camDepthSensor->getAvailableFrameTypes(frameTypes);
-        convertDepthSensorFrameTypesToProtoMsg(frameTypes,
+        for (auto &frame : frameTypes) {
+            aditof::DepthSensorFrameType frameDetails;
+            camDepthSensor->getFrameTypeDetails(frame, frameDetails);
```

Comment:
You need another use-case here as well.

Reply:
Added a use-case for GET_FRAME_TYPE_DETAILS marked with TO DO.